Gemfile.lock

```diff
@@ -22,6 +22,7 @@ GEM
     faraday (0.14.0)
       multipart-post (>= 1.2, < 3)
     ffi (1.9.23-x64-mingw32)
+    ffi (1.9.23-x86-mingw32)
```

Comment:
Okay, I'm not quite sure why the Gemfile is edited in your PR, but I suspect that you may have an x86 installation of Jekyll on your machine, so the bundler simply installed the x86 dependency alongside the x64 dependency? Approved, because I don't think it'll be a problem, and others might have the same situation - next time though, please add a note in your PR summary on why something unrelated like that changed 🙂